modules/apps/object/object-service/src/main/java/com/liferay/object/service/impl/ObjectActionLocalServiceImpl.java

```diff
@@ -402,6 +407,11 @@ public ObjectAction updateObjectAction(
 		objectAction.setActive(active);
 		objectAction.setConditionExpression(conditionExpression);
 		objectAction.setDescription(description);
+
+		if (description.length() > 75) {
+			throw new ObjectActionDescriptionException.MustBe75Characters();
+		}
```

Comment:
@Aquiiiles please move this logic to the new method called `_validateDescription`.

You should call this method after the `_validateActive` and before the `_validateErrorMessage`